cdc/owner.go

```diff
@@ -766,11 +818,10 @@ func (o *Owner) flushChangeFeedInfos(ctx context.Context) error {
 		failpoint.Inject("InjectActualGCSafePoint", func(val failpoint.Value) {
 			actual = uint64(val.(int))
 		})
-
-		if actual > minCheckpointTs {
+		if actual > gcSafePoint {
 			// UpdateServiceGCSafePoint has failed.
-			log.Warn("updating an outdated service safe point", zap.Uint64("checkpoint-ts", minCheckpointTs), zap.Uint64("actual-safepoint", actual))
-
+			// log.Warn("updating an outdated service safe point", zap.Uint64("checkpoint-ts", gcSafePoint), zap.Uint64("actual-safepoint", actual))
+			log.Warn("updating an outdated service safe point", zap.Time("checkpoint-ts", oracle.GetTimeFromTS(gcSafePoint)), zap.Time("actual-safepoint", oracle.GetTimeFromTS(actual)))
```

Comment:
Please remove the comment line. Besides use `zap.Uint64("checkpoint-ts", gcSafePoint), zap.Uint64("actual-safepoint", actual)` is more accurate

Reply:
ok, I've restored it.